com.github.slgobinath.SafeEyes.yaml

```diff
@@ -0,0 +1,99 @@
+id: com.github.slgobinath.SafeEyes
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+command: safeeyes
+
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--system-talk-name=org.freedesktop.DBus" # for tray icon
+  - "--talk-name=org.freedesktop.Notifications" # for 'notification before breaks' plugin
+  - "--talk-name=org.mpris.MediaPlayer2.*" # to be able to pause media player on break
+  # various DE tray support
+  - "--talk-name=com.canonical.indicator.application"
+  - "--talk-name=com.canonical.AppMenu.Registrar"
+  - "--talk-name=org.ayatana.indicator.application"
+  - "--talk-name=org.gnome.Mutter.IdleMonitor"
+  - "--talk-name=org.kde.StatusNotifierWatcher"
+  - "--env=GTK_PATH=/app/lib/gtk-3.0"
+
+cleanup: # Cleanup after the python modules
+  - /cache
+  - /man
+  - /share/aclocal
+  - /share/devhelp
+  - /include
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /lib/*.a
+  - /lib/*.la
+
+modules:
+  - shared-modules/libappindicator/libappindicator-gtk3-introspection-12.10.json # app indicator support
+  - shared-modules/libappindicator/libappindicator-gtk3-12.10.json
+  - libcanberra/libcanberra.json # don't clean /lib/gtk-3.0
```

Comment:
why do you not use the shared-module?

Reply:
After this [commit](https://github.com/flathub/shared-modules/commit/096ab6b0be271f07a42a647b5ffc83808a160e59), installation deletes `/lib/gtk-3.0` too so instead of forking it I'm just using the same without the gtk-3.0 line

Without  `/lib/gtk-3.0`, app will give a warning `Failed to load module "canberra-gtk-module"` probably it's not that important but I didn't check the whole codebase and upstream says it's a requirement

Reply:
this is how all the packages that use it use it.